src/main/java/io/appform/dropwizard/actors/base/UnmanagedPublisher.java

```diff
@@ -54,19 +54,20 @@ public final void publishWithDelay(final Message message, final long delayMillis
         publishWithDelay(message, properties);
     }
 
-    private final void publishWithDelay(final Message message, final AMQP.BasicProperties properties) throws Exception {
+    private void publishWithDelay(final Message message, final AMQP.BasicProperties properties) throws Exception {
```

Comment:
Why is final removed ? Is there a requirement to override this method anywhere ?

Reply:
Since this is private method is cannot be overridden so final is unnecessary here.